### DISCUSSION TITLE
useEffect not firing?

### DISCUSSION THREAD

**Discussion by Firebrand** [vercel/next.js#13084] [Help] [answered]
I'm using "next dev" to test this simple page script. The page runs fine, but the useEffect hook never fires.

```
import {useEffect} from "react"

const TestPage = () => {
    useEffect(() => {
        console.log("Testing");
    });
    return (<div>Test</div>);
}

export default TestPage;
```

**Comment by jensmeindertsma** [comment, marked as answer]
Hello @Firebrand . This could be because you omitted the dependency array that `useEffect` takes as a second argument, [read more here](https://reactjs.org/docs/hooks-effect.html). However, it should work 
fine without. Can you post a repository / more code that I can take a look at?

**Reply by Firebrand** [reply]
Thanks Jens. Your answer jolted me and made me realize that this would have been reported by a lot of people had it been a real bug. 

I was looking at the server console instead of the browser console.   :-\

**Reply by dhavaljardosh** [reply]
Ohh man, I was doing the same thing looking at server console. Thank you

**Reply by leovazquezz1** [reply]
Same

**Reply by namlh023** [reply]
me too
